### PR TITLE
Layout & formatting of example descriptions on FHIR Artefacts page

### PR DESCRIPTION
**Pull request by dbojicic**
Improvement of layout and formatting example descriptions on FHIR Artefacts page.
This PR does not include in scope improving the content of the descriptions.

### CODE REVIEW

**Comment by dt-r** [comment]
@dbojicic this will be reviewed in parallel with the equivalent PR for AU Core. Until that PR is available this is on hold.

File: input/examples/organization-elimbah-medical-center.xml
Comment (dt-r) [inline input/examples/organization-elimbah-medical-center.xml:9]: Insufficient, provide information on the organization example.

File: input/examples/patient-fred-roberts.xml
Comment (dt-r) [inline input/examples/patient-fred-roberts.xml:9]: Insufficient, provide information on the patient example. This project task isn't about better grammar it's about layout information for the reader.

File: input/examples/order-xray-1.xml
Comment (dt-r) [inline input/examples/order-xray-1.xml:8]: Insufficient, provide more information on the example.

File: input/examples/order-fbc-1.xml
Comment (dt-r) [inline input/examples/order-fbc-1.xml:8]: Insufficient provide more information on the example.

File: input/examples/encounter-consultation.xml
Comment (dt-r) [inline input/examples/encounter-consultation.xml:7]: Insufficient, provide more information on the encounter.

File: input/examples/practitioner-guthridge-jarred.xml
Comment (dt-r) [inline input/examples/practitioner-guthridge-jarred.xml:9]: Provide more info.

File: input/examples/organization-elimbah-medical-center.xml
Comment (dt-r) [inline input/examples/organization-elimbah-medical-center.xml:10]: For all AU Core profiles don't say "using the ..." have same text as eRequesting "of ..."

File: input/examples/practitionerrole-guthridge-jarred-gp.xml
Comment (dt-r) [inline input/examples/practitionerrole-guthridge-jarred-gp.xml:9]: Not enough information on the example.

File: input/examples/encounter-consultation.xml
Comment (dt-r) [inline input/examples/encounter-consultation.xml:7]: Don't linebreak Patient. Apply to all example markdown.

**Review by dt-r — CHANGES_REQUESTED** [review]
See inline comments, noting given timeframes (this needs to be merged today) and priorities it may be better to:
* address only consistency and formatting. 
* reframe this PR as layout & formatting rather than substantial content improvements to the descriptions. 
* move the project task into the backlog to come back and improve these descriptions post ballot snapshot. 

I will leave that to your discretion.

File: input/examples/encounter-consultation.xml
Comment (dbojicic) [reply input/examples/encounter-consultation.xml:7]: Changed

File: input/examples/organization-elimbah-medical-center.xml
Comment (dbojicic) [reply input/examples/organization-elimbah-medical-center.xml:10]: Fixed

**Comment by dbojicic** [comment]
Addressed the formatting and consistency issues above. 
Will move the task to the backlog to address post ballot snapshot.

File: input/examples/encounter-consultation.xml
Comment (dt-r) [inline input/examples/encounter-consultation.xml:7]: Missing the Patient for this example

Comment (dt-r) [inline input/examples/encounter-consultation.xml:7]: For all examples, to be consistent with AU Core:
1. Start with "Shows an example of <insert thing here> for the *<insert profile* profile. Patient <insert patient>."

**Review by dt-r — CHANGES_REQUESTED** [review]
See inline comments

File: input/examples/encounter-consultation.xml
Comment (dbojicic) [reply input/examples/encounter-consultation.xml:7]: Added

Comment (dbojicic) [reply input/examples/encounter-consultation.xml:7]: Reformatted

Comment (dt-r) [reply input/examples/encounter-consultation.xml]: Some were, some aren't

File: input/examples/order-xray-1.xml
Comment (dt-r) [inline input/examples/order-xray-1.xml:8]: Reformat this one.

File: input/examples/order-fbc-1.xml
Comment (dt-r) [inline input/examples/order-fbc-1.xml:8]: Reformat this one.

**Review by dt-r — CHANGES_REQUESTED** [review]
See inline comments.

File: input/examples/order-xray-1.xml
Comment (dbojicic) [reply input/examples/order-xray-1.xml]: Fixed.

File: input/examples/order-fbc-1.xml
Comment (dbojicic) [reply input/examples/order-fbc-1.xml]: Fixed.

**Review by dt-r — APPROVED** [review]

**Pull request merged**
